contributors/devel/e2e-tests.md

```diff
@@ -1,6 +1,6 @@
 # End-to-End Testing in Kubernetes
 
-Updated: 5/3/2016
+Updated: 25/10/2017
```

Comment:
Can you just remove this date line? It's not needed when we have git history.

Reply:
Yes,  good catch.